packages/ember-glimmer/lib/component-managers/curly.ts

```diff
@@ -74,26 +91,26 @@ function applyAttributeBindings(element, attributeBindings, component, operation
   }
 }
 
-function tagName(vm) {
+function tagName(vm: any) {
   // tslint:disable-next-line:no-shadowed-variable
   let { tagName } = vm.dynamicScope().view;
 
   return PrimitiveReference.create(tagName === '' ? null : tagName || 'div');
 }
 
-function ariaRole(vm) {
+function ariaRole(vm: any) {
```

Comment:
there is a type exported for this, I think it is called PublicVM

Reply:
Works for the `ariaRole` function, but not `tagName`: `Property 'view' does not exist on type 'DynamicScope'`